custom_components/easee/controller.py

```diff
@@ -214,6 +225,43 @@ async def schedules_async_refresh(self):
 
         _LOGGER.debug("Schedule: %s %s", self.schedule, self.weekly_schedule)
 
+    def schedules_interpret(self, data):
```

Comment:
```suggestion
    def schedules_interpret(self, data):
        """Interpret schedule data."""
```
Add docstring to reduce pylint complaints